make/migrations/postgresql/0006_ng_replication.up.sql

```diff
@@ -12,4 +12,35 @@ CREATE TABLE registry (
  creation_time timestamp default CURRENT_TIMESTAMP,
  update_time timestamp default CURRENT_TIMESTAMP,
  CONSTRAINT unique_registry_name UNIQUE (name)
-);
\ No newline at end of file
+);
+
+create table replication_execution (
```

Comment:
Put all sql to https://github.com/goharbor/harbor/blob/replication_ng/make/migrations/postgresql/0004_1.8.0_schema.up.sql